### PR TITLE
add SOURCE files

### PR DESCRIPTION
**Pull request by escapewindow**
I added these to the top level - not sure if that is ok. It seems to
make sense to me, but I'm open where it goes / what it's named.

### CODE REVIEW

File: beetmoverscript/templates/firefox_candidates.yml [hunk 1]
```diff
@@ -108,6 +108,14 @@ mapping:
       s3_key: {{ platform }}/{{ locale }}/firefox-{{ version }}.tar.bz2.asc
       destinations:
         - {{ platform }}/{{ locale }}/firefox-{{ version }}.tar.bz2.asc
+    SOURCE:
```
Comment (MihaiTabara) [inline beetmoverscript/templates/firefox_candidates.yml:111]: Location sgtm. It should be in the top-level directory from my standpoint.

File: beetmoverscript/templates/firefox_candidates_repacks.yml [hunk 2]
```diff
@@ -47,6 +47,14 @@ mapping:
       s3_key: {{ platform }}/{{ locale }}/firefox-{{ version }}.tar.bz2.asc
       destinations:
         - {{ platform }}/{{ locale }}/firefox-{{ version }}.tar.bz2.asc
+    SOURCE:
```
Comment (MihaiTabara) [inline beetmoverscript/templates/firefox_candidates_repacks.yml]: I don't think this is necessary. This is template for firefox_candidates_repacks which is not the case here. I suppose an en-US beetmover linux/linux64 based job will transfer this, hence this section is not needed.

**Review by MihaiTabara — APPROVED** [review]
Thanks for this. Just a brief comment there. I can merge as is too, more like a nit.

**Comment by escapewindow** [comment]
Removed the firefox_candidates_repacks.yml change.

**Comment by escapewindow** [comment]
Also, we're out of the change freeze, so if you want to land backwards-compatible changes, even if they're not complete, we can do that before we're done.

**Comment by MihaiTabara** [comment]
@escapewindow  For now, I've merged this and deployed a 3.3.0-dev10 so that you can rerun your task to see the artifact copied over. I plan to work a bit on Sunday to finish up checksums and make some progress on balrog so I'll look into prepping the PR for a production rollout. Good idea 💡

**Pull request merged**